### PR TITLE
fix(farming-calc) Add timezone and refine roll token messaging

### PR DESCRIPTION
**Pull request by nicholaspai**
Responding to user feedback:

Screen Shot 2020-09-27 at 01 19 20

### CODE REVIEW

File: features/yield/FarmingCalculator.tsx [hunk 1]
```diff
@@ -65,12 +65,9 @@ const FarmingCalculator = () => {
   );
   const rollToPool = getPoolDataForToken(rollToTokenAddress);
   const rollStartDate = rollFromTokenObj && rollFromTokenObj.rollStartDate;
-  const rollFromTokenName = isRollToToken
-    ? rollFromTokenObj && rollFromTokenObj.rollFromTokenName
-    : rollFromTokenObj && rollFromTokenObj.rollToTokenName;
-  const rollToTokenName = !isRollToToken
-    ? rollFromTokenObj && rollFromTokenObj.rollFromTokenName
-    : rollFromTokenObj && rollFromTokenObj.rollToTokenName;
+  const rollFromTokenName =
```
Comment (nicholaspai) [inline features/yield/FarmingCalculator.tsx:68]: @chrismaree what was the reasoning behind the `isRollToToken` logic? It seemed to be showing the wrong `rollToTokenName` when selecting the `rollFromToken` like the OCT tokens.

Comment (chrismaree) [reply features/yield/FarmingCalculator.tsx:68]: The logic was to ensure that we always had the right combination of `from` and `to` token names during the roll.

**Review by chrismaree — APPROVED** [review]
LGTM.

File: features/yield/FarmingCalculator.tsx (hunk 1 above)
Comment (nicholaspai) [reply features/yield/FarmingCalculator.tsx:68]: This actually worked mid-roll

Comment (nicholaspai) [reply features/yield/FarmingCalculator.tsx:68]: I tested it out pre-roll, post-roll, mid-roll. I think it probably reverses if you don't supply the `rollToToken` heh

**Pull request merged**